package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@aws/clickstream-web",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
why not using release workflow to bump the version?

Reply:
This will be fixed in another PR